smssync/src/main/java/org/addhen/smssync/presentation/service/ServiceControl.java

```diff
@@ -207,13 +207,8 @@ public boolean isConnected() {
         return Utility.isConnected(mContext);
     }
 
-    public enum SchedulerInstance {
```

Comment:
> More thread-safety this way.

Surely something's thread-safe or it's not - the area between the two is scary (and _not_ thread-safe)!